src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -117,6 +118,8 @@ private async Task<ImmutableArray<AddImportFixData>> GetFixesInCurrentProcessAsy
                             if (result.Count > maxResults)
                                 break;
                         }
+
+                        GC.KeepAlive(semanticModel);
```

Comment:
something i noticed.  we get a semantic mdoel above, but also within some of the nested calls from this method.  so by doing an explicit keepalive here, we ensure the inner calls are always cost-free.